### DISCUSSION TITLE
Force Constants for SCPH calculation

### DISCUSSION THREAD

**Discussion by LR-siit** [ttadano/alamode#28] [Q&A]
Can we use the cubic and quartic force constants extracted from the ALM package for the SCPH calculation? Since in the hands-on tutorial package and the paper,  you have extracted the anharmonic force constants using the compressive sensing approach and AIMD dataset with random displacements, I was confused since my results varying largely even if I follow the same steps as you mentioned. I used this method for Boron Arsenide which calculated the FCs from ALM and used it for SCPH to calculate lattice thermal conductivity.

And Is ther anyway that we can convert ALM FC file into QuantumESPRESSO format?
If you can provide some suggestion, I am much grateful.
Thank you

**Comment by ttadano** [comment]
> Can we use the cubic and quartic force constants extracted from the ALM package for the SCPH calculation?

Yes.

Could you describe more details about your problem? What do you mean by "result varying largely"?

> And Is ther anyway that we can convert ALM FC file into QuantumESPRESSO format?

We don't provide a script for converting ALM FC file to QE fc. 
If you need force constants in the QE fc format, I would suggest using ph.x-->q2r.x from scratch.
In most cases, this approach is far more efficient than the supercell method for calculating harmonic FCs.
